.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
It's generally better to keep IDE (and OS)-specific files out of the project's concerns, otherwise you also end up with entries like `.DS_Store` and `nbproject/*` added to the repository as people come along with new tools or operating systems.

Personally I use a global `.gitignore` file which contains `idea/` and `.DS_Store`, which then means those two files are _never_ added to any project I work on. More info here: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
@nealio82 Thanks for `~/.gitignore_global` concept information.  I was revert that change.